addons/my-ocular/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Show ocular.jeffalo.net statuses and post reactions",
-  "description": "Shows ocular statuses and reactions. Uses jeffalo.net to obtain data.",
+  "name": "ocular.jeffalo.net statuses and reactions",
+  "description": "Shows ocular statuses and post reactions on the forums. Uses my-ocular.jeffalo.net to obtain data.",
```

Comment:
No need to add subdomains

Reply:
I wanted searching for "Show ocular.jeffalo.net statuses and post reactions" to return this addon. Without this change it didn't because neither the title nor the description contained all keywords.

Reply:
Maybe a new field that just dumps keywords?

Reply:
@apple502j We would need to find a way to search in all fields at once first. And ideally also a way to make translators able to increase or decrease the number of keywords.

Reply:
The term my-ocular should not be used. Use ocular instead. Cc @jeffalo

Reply:
@RedGuy12 The API from which this addon gets data is located at my-ocular.jeffalo.net. The ideal solution for searching by old addon names would be searching in all fields at once, but fuse.js doesn't do that by default and I'm not sure how difficult implementing that would be.

Reply:
The name's "ocular," so can we just do that word in replacement with the subdomain? It should've been mentioned on the description. Small caps should be fine since it's a name, just like iPhone.

Reply:
ocular is the user facing name. And you can view ocular statuses on profiles (outside of forums) with this addon

Reply:
I'll change the name of this addon to "ocular statuses and reactions" and description to "Shows ocular statuses and forum post reactions. Uses my-ocular.jeffalo.net to obtain data.".